webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -678,6 +681,12 @@ var userOverride = {
                       .removeClass('vis')
                       .addClass('n-vis')
                 : '';
+            document
+            .getElementById('chat-popup-widget-container')
+            .firstChild.removeEventListener(
+                'click',
+                KommunicateUI.captureGreetingMessageClick
```

Comment:
this event also seem to be twice

Reply:
we are attaching click event when we found the greeting message `div `in DOM and removing it once the chat widget gets open.